pkg/soperator/utils.go

```diff
@@ -177,3 +175,12 @@ func GetOldPrometheusSpec(restConfig *rest.Config, kubeClient *kubernetes.Client
 	return &prometheusSpec, nil
 
 }
+
+// GetEnvironment returns the Environment type
+func GetEnvironment(restConfig *rest.Config) Environment {
```

Comment:
environment is overloaded here.  Suggest something like GetClusterType

Reply:
Good point. I changed it to ClusterType instead of Environment everywhere